app/src/renderer/components/staking/TabMyDelegations.vue

```diff
@@ -99,6 +99,12 @@ export default {
               transaction.tx.value.msg[0].value.validator_addr
             ]
         }))
+  },
+  async mounted() {
+    this.$store.dispatch(`getTotalRewards`)
+    this.yourValidators.forEach(validator => {
+      this.$store.dispatch(`getRewardsFromValidator`, validator.operator_address)
```

Comment:
this is good for now. might be an issue if a user has too many validators... maybe a good candidate for a new special endpoint on the REST API. 

@alexanderbez 

maybe the endpoint `/distribution/delegators/cosmos1address/rewards` could return the data per validator instead of as a single total value? just an idea.